public/css/addApplicant.css

```diff
@@ -0,0 +1,3 @@
+label {
```

Comment:
is it really necessary for this to have its own file?!

Reply:
will remove. was from last cohort. I just added the line at the end of the file while I was checking through them all as github highlighted it.